### DISCUSSION TITLE
Publishing notes

### DISCUSSION THREAD

**Discussion by kleters** [standardnotes/forum#3405] [General]
Hi everybody,

Let's say i want to publish some of my notes (that are organized in folders and subfolders) and give readers a similar user-experience as if they were using standardnotes. I do not want to create a blog, so I guess listed is not the right choice. 
The most convenient way I could think of is just uploading the folders with the .txt-files from my standardnotes backup to a hoster that displays the notes accordingly. Do you know any?

Kind regards,
kleters

**Comment by effieeee** [comment]
I'm not aware of any service that can replicate the look and feel of Standard Notes on a public platform per se, but maybe there are other blog sites that can allow you to organize your published posts (notes) in the same way?
